File: 16-fizzbuzz/fizzbuzz.py
# Python program to print FizzBuzz
# Created by M.V.Harish Kumar on 15-01-2024

n = int(input("Enter no. of terms: "))
for i in range(1, n+1):
    if i % 15 == 0:
        print("FizzBuzz")
    elif i % 3 == 0:
        print("Fizz")
    elif i % 5 == 0:
        print("Buzz")
    else:
        print(i)

File: 16-fizzbuzz/fizzbuzz.rb
# FizzBuzz program in Ruby
# Created by M.V.Harish Kumar on 01-02-2025

print "Enter no. of terms: "
n = gets.to_i

(1..n).each do |i|
  if i % 15 == 0
    puts "FizzBuzz"
  elsif i % 5 == 0
    puts "Buzz"
  elsif i % 3 == 0
    puts "Fizz"
  else
    puts i
  end
end
